README.md

```diff
@@ -1,2 +1,4 @@
 # signal-strength-logging
 Logs signal strength.
+
+This script assumes `minicom` is installed on to communicate wtih the modem.
```

Comment:
Nit: We like to consolidate all of our repository information into confluence pages. You can see this in the other repositories (e.g. [computer-vision-python](https://github.com/UWARG/computer-vision-python)). Can you create a confluence doc and add this information there. Then link the confluence doc in the `README.md` similar to how computer-vision-python does it. 

Also, you don't need to have the confluence document finished. You can create one for now and write documentation for your script later :).